ci/build-openbsd.sh

```diff
@@ -19,16 +21,13 @@ libswresample = libswresample_dep
 libswscale = libswscale_dep
 EOF
 
-meson setup build \
-    -Dffmpeg:vulkan=auto \
-    -Dlibmpv=true \
-    -Dlua=enabled \
-    -Dopenal=enabled \
-    -Dpulse=enabled \
-    -Dtests=true \
-    -Dvulkan=enabled \
-    -Ddvdnav=enabled \
-    -Dcdda=enabled
-
+meson setup build $common_args \
+  -Dffmpeg:vulkan=auto \
+  -Dffmpeg:werror=false \
+  -Dlua=enabled \
+  -Dopenal=enabled \
+  -Dpulse=enabled \
+  -Dvulkan=enabled \
+  -Ddvdnav=enabled
```

Comment:
Instead of removing dvdnav/cdda I would remove werror for this build.

Reply:
This one currently doesn't have werror at all. I thought removing cdda would be better.